Jenkinsfile

```diff
@@ -87,7 +87,7 @@ pipeline {
                         cp debian/changelog ${WORKSPACE}/debian/changelog-${SERIES_VERSION}
                         sed -i "s/${PKG_VERSION}/${NEW_PKG_VERSION}/" ${WORKSPACE}/debian/changelog-${SERIES_VERSION}
                         dpkg-source -l${WORKSPACE}/debian/changelog-${SERIES_VERSION} -b .
-                        sbuild --nolog --verbose --dist=${BUILD_SERIES} --no-run-lintian --append-to-version=~${SERIES_VERSION}  ../ubuntu-advantage-tools*${NEW_PKG_VERSION}*dsc
+                        sbuild --resolve-alternatives --nolog --verbose --dist=${BUILD_SERIES} --no-run-lintian --append-to-version=~${SERIES_VERSION}  ../ubuntu-advantage-tools*${NEW_PKG_VERSION}*dsc
```

Comment:
The `golang-1.10 | golang` in Build-Depends only works when sbuild gets this flag. `sbuild-it` passes this flag by default but I'm not sure if its safe to rely on it. Thoughts?

Reply:
As discussed in this bug: https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1926949, it is okay to rely on --resolve-alternatives.